TESTPG5.NSP.txt

```diff
@@ -0,0 +1,14 @@
+DEFINE DATA LOCAL
+01 #CNT (N2)
+01 #CNT1 (N2)
+01 #CNT2 (N2)
+01 #CNT3 (N3)
+01 #CNT4 (N3)
+END-DEFINE
+ 
+#CNT1 := 2
+#CNT2 := 2
+#CNT3 := #CNT1 * #CNT2
+#CNT4 := #CNT3
+WRITE 'COMPUTED VALUE' #CNT4

```

Comment:
Please include all local variables in WRITE statement

Reply:
Fixed ..